src/confread.c

```diff
@@ -51,40 +51,42 @@
     #define MAX_TOKEN_LENGTH  30    // Default max token length
 #endif
 
-FILE            *confFilePtr;       // File handle pointer
-char            *iBuffer;           // Inputbuffer for reading...
-unsigned int    bufPtr;             // Buffer position pointer.
-unsigned int    readSize;           // Number of bytes in buffer after last read...
-char    cToken[MAX_TOKEN_LENGTH];   // Token buffer...
-short   validToken;
+static struct {
+    FILE            *confFilePtr;       // File handle pointer
+    char            *iBuffer;           // Inputbuffer for reading...
+    unsigned int    bufPtr;             // Buffer position pointer.
+    unsigned int    readSize;           // Number of bytes in buffer after last read...
+    char    cToken[MAX_TOKEN_LENGTH];   // Token buffer...
+    short   validToken;
+} conf;
```

Comment:
Is putting all variables into struct really needed? They are private for confread.c file, so `static` should be enough.

Reply:
ok